src/main.py

```diff
@@ -285,8 +292,8 @@ def parse_cli_args(cli_args=None):
         action="store_true",
     )
     parser.add_argument(
-        "--validate-only",
-        help="Do not apply updates, only check if metadata is valid",
+        "--validate",
+        help="Set exit code based on errors amount",
```

Comment:
Why is this conditional? Why not always exit with a defined error code in this case? (Sincere question.)

```
         help="Exit with non-zero status if errors are found while checking",
```

Reply:
Currently we set exit code based on amount of outdated data - 0 if all up to date and 1 if something is outdated. I suppose we should retain this default behaviour?

Reply:
Maybe we should combine non-zero exit code? E.g. `1` if outdated, `2` if errors, `3` if both.

Reply:
Now this is unconditional.